src/Jackalope/Version/VersionHandler.php

```diff
@@ -174,4 +221,40 @@ public function checkoutItem($path)
 
         $this->session->save();
     }
+
+    /**
+     * Creates a frozen node
```

Comment:
can you be more explicit here? "Attaches a nt:frozenNode copy of $node to $versionNode"